ai2thor/video_controller.py

```diff
@@ -287,9 +262,14 @@ def AbsoluteCameraAnimation(self,
                                  'y': p0['y'] + (py - p0['y']) / frames * i,
                                  'z': p0['z'] + (pz - p0['z']) / frames * i},
                 }
-                if not (smartSkybox and maxY > p0['y'] + (py - p0['y']) / frames * i):
-                    kwargs['skyboxColor'] = 'black'
-                yield self.step(**kwargs)
+
+            # enables linear animation changes to the camera FOV
+            if FOVstart != None and FOVend != None:
```

Comment:
im not sure if/where this code is used, but this should work to update fov now

Reply:
Can you include @mattdeitke in the review? he wrote this video controller to make sure the functionality is the same

Reply:
done